### PR TITLE
update project models & test

### PR DESCRIPTION
**Pull request by o8151814**
refactor project test.
fix create_project error

### CODE REVIEW

File: buildbuild/projects/tests/test_project_team.py [hunk 1]
```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from projects.models import Project
+from teams.models import Team
+from django.core.exceptions import ObjectDoesNotExist
```
Comment (dobestan) [inline buildbuild/projects/tests/test_project_team.py]: why `ObjectDoesNotExist` exception is imported?

File: buildbuild/projects/models.py [hunk 2]
```diff
@@ -1,16 +1,19 @@
 from django.db import models
 from django.core.exceptions import ValidationError
-from teams.models import Team
+from teams.models import Team, TeamManager
```
Comment (dobestan) [inline buildbuild/projects/models.py]: why `TeamManager` is imported?

Comment (dobestan) [inline buildbuild/projects/models.py]: and i think `from teams.models import Team, TeamManager` is not needed. ( both )

```
def create_project(self, name, team):
...
```

i think ...
- `team` is just a parameter name.
- `projects/models.py` should not know what is `Team` and `TeamManager`, but only `project should belongs to team` ( as parameter name and instance type )

Comment (o8151814) [inline buildbuild/projects/models.py]: yes.... Sorru

File: buildbuild/projects/tests/test_project_team.py (hunk 1 above)
Comment (o8151814) [inline buildbuild/projects/tests/test_project_team.py]: it is my mistake

File: buildbuild/projects/models.py (hunk 2 above)
Comment (dobestan) [inline buildbuild/projects/models.py]: @o8151814 no! you're awesome!

File: buildbuild/projects/tests/test_project_team.py (hunk 1 above)
Comment (o8151814) [inline buildbuild/projects/tests/test_project_team.py]: i wanted to add validate-team. but itis not useful

File: buildbuild/projects/models.py (hunk 2 above)
Comment (o8151814) [inline buildbuild/projects/models.py]: thank you ♬

**Comment by o8151814** [comment]
Yes, I will fix that.

**Pull request merged**